oemof_b3/model/component_attrs_update.yml

```diff
@@ -54,6 +72,26 @@ electricity-electrolyzer:
   defaults:
     output_parameters: "{}"
 
+electricity-export:
+  carrier: electricity
+  tech: export
+  type: excess
+  foreign_keys:
+    bus: electricity
+  defaults:
+    marginal_cost: 0
```

Comment:
I suggest that we do not set the marginal cost as a default here. IMO it will be nonzero and defined in the scalar data.

Reply:
:+1:  I have removed it from all import/export components.